rmgpy/molecule/group.py

```diff
@@ -1368,20 +1368,22 @@ def sortByConnectivity(atomList):
             Returns: a sorted list of atoms where each atom is connected to a previous
```

Comment:
Looked into it a little bit, but I don't think I can without changing the whole code structure. Because `sortByConnectivity()` is a function underneath `addImplicitBenzene() ` , I cannot seem to access sortByConnectivity except in the scope of addImplicitBenzene.